src/Components/ArticleDetailPage.jsx

```diff
@@ -50,7 +55,20 @@ function ArticleDetailPage() {
           <p>{article.body}</p>
           <p className='votes'>{article.votes} votes</p>
           <p className='comments'>{article.comment_count} comments</p>
-          <CommentList articleId={article_id}/>    
+          
+          {user ? ( 
+          
+            <CommentForm articleId={article_id} onCommentPosted={() => fetchArticle()}/>
```

Comment:
We don't want to refetch the article when we post a comment as this doesn't take into account other users posting comments.